### DISCUSSION TITLE
[PyData Manaus] Apoio da APyB

### DISCUSSION THREAD

**Discussion by teodororo** [apyb/comunidade#174] [Pergunta]
Olá, tudo bem?

Faço parte do PyData Manaus, uma comunidade de ciência de dados de Manaus, Amazonas. O PyData é um programa educacional criado pela NumFOCUS, uma organização sem fins lucrativos estadunidense. Pela falta de representação jurídica nacional, estamos enfrentando dificuldades para fechar patrocínios. Nesse sentido, acreditamos que nos afiliarmos à Associação Python Brasil possa ser a solução, pois resolveria a falta de CNPJ. Não obtivemos retorno da NumFOCUS para saber se havia algum impeditivo.

Dessa forma, ainda que eu tenha preenchido o Formulário de Solicitação de Apoio a Eventos, gostaria de deixar essa situação clara e transparente para evitar mal-entendidos. Isso é, é possível usarmos a infraestrutura da APyB mesmo com a relação com a NumFOCUS?

Destaco que a comunidade só voltou à ativa após a divulgação do evento durante uma _lightning talk_ na Python Norte 2024, e a sugestão de buscar ajuda da APyB foi feita pelos organizadores desse evento.

Agradeço desde já pela atenção e fico à disposição para fornecer mais informações ou esclarecer quaisquer dúvidas. :grinning:

cc: @jailsonpj @Diegovisk

**Comment by jailsonpj** [comment]
@cecivieira

**Comment by tiidadavena** [comment]
Olá, Giovanna e demais representantes!

Confirmamos o registro de solicitação de apoio para infraestrutura de eventos cujo código é o `pydatamanaus7`. A APyB também fornece ajuda para eventos pythonistas fora do eixo da PSF, como Django Girls, que está mais diretamente ligada a Django Software Foundation.

Para mais dúvidas sobre como solicitar emissão de Nota Fiscal e recebimento de patrocínio do exterior, por exemplo, basta acessar o nosso guia disponível através [deste link](https://apyb.python.org.br/apyb/faq/).

**Reply by tiidadavena** [reply]
E muito sucesso para a toda a comunidade PyData Manaus!

**Reply by teodororo** [reply]
Olá, bom dia. Muitíssimo obrigada pela resposta e aceite. :heart: 
Sendo assim, já vamos incluir a logo da APyB em tudo que for possível. :smiley: 
De novo, obrigada.
